clearwater-queue-manager.root/usr/share/clearwater/clearwater-queue-manager/scripts/modify_nodes_in_queue.py

```diff
@@ -59,39 +59,34 @@ def make_key(site, clearwater_key, queue_key):
 
 if operation == "add":
     logging.debug("Adding %s to queue to restart" % (local_ip + "-" + node_type))
-
```

Comment:
I'm not a fan of these line changes - I think how they were before is more common